cluerr/errcore.go

```diff
@@ -70,7 +69,13 @@ func (ec *ErrCore) String() string {
 // stringer handles all the fancy formatting of an errorCore.
 func (ec *ErrCore) stringer(fancy bool) string {
 	sep := ", "
-	ls := strings.Join(maps.Keys(ec.Labels), sep)
+	keys := []string{}
+
+	for k := range ec.Labels {
+		keys = append(keys, k)
+	}
+
+	ls := strings.Join(keys, sep)
```

Comment:
oof, now I want a `strings.Join(seq iter.Seq[string], separator string)` call. Since we don't have that, you could also do something like the below if you want a one-liner

```suggestion
	ls := strings.Join(slices.Collect(maps.Keys(ec.Labels)), sep)
```

Reply:
I'll save this for a migration to a helper method in a separate PR.  I'd definitely like a `strings.Collect` as well.  But until then a helper will suffice.